litellm/utils.py

```diff
@@ -4192,7 +4192,14 @@ def _map_and_modify_arg(supported_params: dict, provider: str, model: str):
                 optional_params["stop_sequences"] = stop
         if max_tokens is not None:
             optional_params["max_output_tokens"] = max_tokens
-    elif custom_llm_provider == "vertex_ai":
+    elif custom_llm_provider == "vertex_ai" and model in (
+        litellm.vertex_chat_models
```

Comment:
can we not do this?  I can see a regression occuring for this very easily in the future 

lets stick to the convention of only checking `custom_llm_provider == "vertex_ai":`

Reply:
nvm sav discord convo